commerce-payment-method-mercanet/src/main/java/com/liferay/commerce/payment/method/mercanet/internal/constants/MercanetCommercePaymentMethodConstants.java

```diff
@@ -28,6 +28,8 @@
 		StringUtil.toLowerCase(Environment.SIMU.name())
 	};
 
+	public static final String EUR_CURRENCY_CODE = "EUR";
```

Comment:
Suuuuuper minor detail here, but I think it makes sense to call this `CURRENCY_CODE` rather than `EUR_CURRENCY_CODE` -- if I understand correctly, we're using this because Mercanet operates in `EUR`. If for some crazy reason Mercanet changed their default currency, then `CURRENCY_CODE` would still make sense, but `EUR_CURRENCY_CODE` wouldn't; but ideally if this were to happen, we would just want to change the value, not the name of the constant. Just a slight semantic nuance, but I think it's worth mentioning.